### PR TITLE
CrowdStrike Falcon Sandbox v2

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/etc/issues/10860 https://github.com/demisto/etc/issues/10721 

Includes limited version integration which will include out of the box instance - from version 3.6.1

### CODE REVIEW

**Comment by Itay4** [comment]
Documentation: https://github.com/demisto/etc/issues/11519

File: TestPlaybooks/playbook-VxStream_Test.yml [hunk 1]
```diff
@@ -1,21 +1,22 @@
 id: VxStream Test
-version: 11
+version: 18
```
Comment (anara123) [inline TestPlaybooks/playbook-VxStream_Test.yml]: change to -1

File: TestPlaybooks/playbook-VxStream_Test.yml [hunk 2]
```diff
@@ -68,9 +70,12 @@ tasks:
...
       - "3"
     scriptarguments:
-      expectedValue: ""
-      fields: ID,description,architecture,VMs_total,VMs_busy,analysisMode,groupicon
-      path: VX.Environment
+      expectedValue: {}
+      fields:
+        simple: ID,description,architecture,VMs_total,VMs_busy,analysisMode,groupicon
+      path:
+        simple: VX.Environment
```
Comment (anara123) [inline TestPlaybooks/playbook-VxStream_Test.yml]: change context `VX.Environment` is not good

**Comment by dem-evgeny** [comment]
Please don't forget to add outputs to these commands:
vx-detonate-file
crowdstrike-detonate-file
thanks

**Pull request merged**